MdeModulePkg/MdeModulePkg.mu.yaml

```diff
@@ -17,7 +17,8 @@
 
             # Not supported without StandaloneMmPkg.
             "MdeModulePkg/Universal/FaultTolerantWriteDxe/FaultTolerantWriteStandaloneMm.inf",
-            "MdeModulePkg/Universal/Variable/RuntimeDxe/VariableStandaloneMm.inf"
+            "MdeModulePkg/Universal/Variable/RuntimeDxe/VariableStandaloneMm.inf",
+            "MdeModulePkg/Universal/Disk/CdExpressPei/CdExpressPei.inf"
```

Comment:
this doesn't match the comment and should probably be removed as it seems like you got it to work.